app/models/filter.rb

```diff
@@ -169,10 +173,10 @@ def build_taxonomy_search
     orgs = build_taxonomy_search_string('organization')
     locs = build_taxonomy_search_string('location')
 
-    orgs = [] if !granular? || !resource_class.allows_organization_filtering?
-    locs = [] if !granular? || !resource_class.allows_location_filtering?
+    orgs = '' unless resource_taxable_by_organization?
```

Comment:
If it's not taxable, how expensive is the call to `build_taxonomy_search_string` then? Should it rather be
```ruby
orgs = if resource_taxable_by_organization?
  build_taxonomy_search_string('organization')
else
  ''
end
```

(same for locs)

Reply:
probably not significant, but even the code looks cleaner so :+1: